packages/web-app-files/src/components/SideBar/Shares/FileShares.vue

```diff
@@ -24,45 +24,69 @@
           data-testid="files-collaborators-no-reshare-permissions-message"
           v-text="noResharePermsMessage"
         />
-        <template v-if="$_ownerAsCollaborator">
-          <p id="original-sharing-user" v-translate class="oc-invisible-sr">File owner</p>
-          <show-collaborator
-            :collaborator="$_ownerAsCollaborator"
-            aria-describedby="original-sharing-user"
-          />
-          <hr />
-          <show-collaborator :collaborator="$_currentUserAsCollaborator" />
-        </template>
-        <template v-else>
-          <p id="collaborator-as-fileowner" v-translate class="oc-invisible-sr">
-            You are the file owner
-          </p>
-          <show-collaborator
-            :collaborator="$_currentUserAsCollaborator"
-            aria-describedby="collaborator-as-fileowner"
-          />
-        </template>
-        <hr v-if="collaborators.length > 0" class="oc-mt-s oc-mb-s" />
-        <transition-group
-          id="files-collaborators-list"
-          class="uk-list uk-list-divider uk-overflow-hidden oc-m-rm"
-          :enter-active-class="$_transitionGroupEnter"
-          :leave-active-class="$_transitionGroupLeave"
-          name="custom-classes-transition"
-          tag="ul"
-          :aria-label="$gettext('Share receivers')"
-        >
-          <li v-for="collaborator in collaborators" :key="collaborator.key">
+        <div class="avatars-wrapper">
+          <h2 v-if="hasSharees" class="shared-with-label" v-text="sharedWithLabel" />
```

Comment:
Please make this a `h4`. We broke the `h2` and `h3` in the sidebar and I'll fix that in a separate PR. But inside of the panel the first available h tag is `h4`.

Reply:
Opening and closing the full list lets the `h`-tag jump vertically a little bit, because the oc-icon has another height than the avatars.